### ISSUE TITLE
Junit - Not possible to select Plug-in Junit tests

### ISSUE THREAD

**Issue by vogella** [angelozerr/jdt-codemining#5]
If I press "Run" I cannot select to run a plug-in JUnit test. 

![selection_260](https://user-images.githubusercontent.com/139910/37827967-c45e2240-2e99-11e8-844b-59c12db31989.png)

Not sure how this can be better handled.

**Comment by angelozerr** [comment]
Indeed, we should think how to manage that. Thank's for having created this issue!

**Comment by angelozerr** [comment]
@vogella have you some idea about UI to fix that?

**Comment by angelozerr** [comment]
@vogella have you some idea about UI to fix that?
